lib/rules/unit-no-unknown/index.js

```diff
@@ -18,6 +19,8 @@ const messages = ruleMessages(ruleName, {
 	rejected: (unit) => `Unexpected unknown unit "${unit}"`,
 });
 
+const mapPropertyNameIndexOffset = 4;
```

Comment:
Could you explain what this variable for, please? I don't understand from the following.

Reply:
The map property name (in map cleared from comments & spaces) always has index that being divided by 4 gives 0 (e.g. 0, 4, 8 etc).
The naming of this `const` definitely has to be changed since it's not clear. Any suggestions? Or may be comment would be enough?

Reply:
I think comment in the code would be enough.

Reply:
Added comment with explanation.